recipes/deepdirect/meta.yaml

```diff
@@ -1,39 +1,35 @@
 {% set name = "deepdirect" %}
-{% set version = "0.2.4" %}
+{% set version = "0.2.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepdirect-{{ version }}.tar.gz
-  sha256: 10feba8b3b52fbb274bdd0bdf9e8ae75df6669029d359a2ba9fae51af53a2faa
+  sha256: ab41f3f53161193146681d477f2412ebdb663bbf76c350e195a05efbb9f032dc
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
   run_exports:
     - {{ pin_subpackage('deepdirect', max_pin="x.x") }}
-
+    
 requirements:
   host:
-    - python
+    - python >=3.8
     - setuptools
     - wheel
     - pip
   run:
-    - python
+    - python >=3.8
     - tensorflow >=2.4.0
     - keras >=2.4.0
-    - colorlog ==6.7.0
-    - tqdm >=4.64.1
 
 test:
   imports:
     - deepdirect
-  commands:
-    - deepdirect -v
```

Comment:
Was the commandline option removed in this latest version?

Reply:
Yes, I may forget to remove these. Could you please help me remove these two lines? Thanks so much.

Reply:
I removed all the command line options in this version because there is an issue with the dependencies when installing. So, I tried to remove useless dependencies and also removed the command line options.

Reply:
Thanks so much for the help.